src/screens/FeedScreen/RenderList.js

```diff
@@ -176,6 +177,16 @@ const RenderListFeed = (props) => {
     }
   };
 
+  const latestComment = React.useMemo(() => {
+    if (!item?.latest_reactions?.comment) return {};
+
+    const sortedComment = item?.latest_reactions?.comment?.sort(
+      (a, b) => moment(b.created_at).valueOf() - moment(a.created_at).valueOf()
+    );
+
+    return sortedComment;
+  }, [item]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `React.useMemo` here is a good practice to avoid unnecessary computations. However, the returned value should be the latest comment instead of the sorted array of comments. Also, the sorting function can be simplified by using `differenceInMilliseconds` from `date-fns`.
```diff
-    const sortedComment = item?.latest_reactions?.comment?.sort(
-      (a, b) => moment(b.created_at).valueOf() - moment(a.created_at).valueOf()
-    );
-
-    return sortedComment;
+    const sortedComment = [...item?.latest_reactions?.comment]?.sort(
+      (a, b) => differenceInMilliseconds(new Date(b.created_at), new Date(a.created_at))
+    );
+
+    return sortedComment[0];
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->